packages/ui-buttons/src/BaseButton/props.ts

```diff
@@ -152,13 +152,12 @@ type BaseButtonOwnProps = {
   renderIcon?: React.ReactNode | (() => React.ReactNode)
 
   // Deprecated `string` tabIndex type
-  // TODO: remove string type in v9
   /**
    * Specifies the tabindex of the `Button`.
    *
    * (`string` type is __deprecated__, use `number`)
    */
-  tabIndex?: number | string
+  tabIndex?: number | string // TODO: remove string type in v9
```

Comment:
I only did these 2 changes so I can add a 'feat' commit, forgot to include in the last release.